src/main.rs

```diff
@@ -249,6 +249,39 @@ pub(crate) async fn run_encrypted_payload(
         }
     }
 
+    // Set execution permission for listed revocation actions
```

Comment:
I think it is better to do this earlier. In the normal deployment, the revocation actions will be installed and stay there untouched until something that requires action happens. Then it would be a bad surprise if they don't work because of permissions that can't be set.